modules/ROOT/pages/batch-filters-and-batch-aggregator.adoc

```diff
@@ -189,49 +183,7 @@ You can carry out the same result as the example above by specifying an arbitrar
 </batch:job>
 ----
 
-Using random access, you can change a record's payload at any given index position in the commit block.
-
-==== Considerations for Defining a Block Size
```

Comment:
MOVED, not edited: This entire section was just moved to a new location on the page without edits:
==== Considerations for Defining a Block Size